index.yaml

```diff
@@ -73,6 +73,13 @@ distributions:
     source_builds:
       default: melodic/source-build.yaml
       ds: melodic/source-stretch-build.yaml
+  noetic:
+    ci_builds:
+      bionic-python2: noetic/ci-bionic-python2.yaml
+      bionic-python3: noetic/ci-bionic-python3.yaml
+    notification_emails:
+    - ros-buildfarm-noetic@googlegroups.com
```

Comment:
@sloretz Can you please create this Google group with the same configuration / members as it exists for the other distros.

Reply:
Group created: https://groups.google.com/forum/#!forum/ros-buildfarm-noetic